eogrow/pipelines/byoc.py

```diff
@@ -27,25 +28,40 @@
 )
 
 LOGGER = logging.getLogger(__name__)
+TIMESTAMP_FORMAT = "%Y-%m-%dT%H-%M-%S"  # TODO: import from .export_maps once MRs are synced
 
 
 class IngestByocTilesPipeline(Pipeline):
     """Ingests .tiff files to a BYOC collection. Based on `ExportMapsPipeline` output for timeless features."""
 
     class Schema(Pipeline.Schema):
         byoc_tile_folder_key: str
-        file_glob_pattern: str = Field("**/UTM_*/*.tiff", description="Pattern used to obtain the TIFF files to use")
+        file_glob_pattern: str = Field("**/*.tiff", description="Pattern used to obtain the TIFF files to use")
 
         new_collection_name: Optional[str] = Field(description="Used for defining a new BYOC collection.")
         existing_collection: Optional[DataCollection] = Field(description="Used when updating and reingesting.")
         _parse_byoc_collection = optional_field_validator("existing_collection", parse_data_collection, pre=True)
         _ensure_exclusion = ensure_exactly_one_defined("new_collection_name", "existing_collection")
 
-        sensing_time: datetime.datetime = Field(description="Sensing time (ISO format) added to BYOC tiles.")
+        is_temporal: bool = Field(
+            False,
+            description=(
+                "If the BYOC is marked as temporal the pipeline will assume that the direct parent folder of a TIFF"
+                " contains the sensing time, i.e.  filesystem structure follows that used by `ExportMapsPipeline`."
+            ),
+        )
+        sensing_time: Optional[datetime.datetime] = Field(
+            None, description="Sensing time (ISO format) added to BYOC tiles. Only used for timeless collections."
+        )
 
         @validator("sensing_time", pre=True)
-        def _parse_sensing_time(cls, value: str) -> datetime.datetime:
-            return datetime.datetime.fromisoformat(value)
+        def _parse_sensing_time(cls, value: Optional[str], values: Dict[str, object]) -> Optional[datetime.datetime]:
+            is_temporal = values["is_temporal"]
+            if is_temporal and value is None:
+                return None
+            if not is_temporal and value is not None:
+                return datetime.datetime.fromisoformat(value)
```

Comment:
does this allow providing just `YYYY-MM-DD`? If not, that would be quite convenient.

Reply:
it does, see [here](https://github.com/sentinel-hub/eo-grow/blob/develop/tests/test_config_files/byoc/ingest_lulc.json)